app/controllers/users/access_token_controller.rb

```diff
@@ -0,0 +1,70 @@
+module Users
+  class AccessTokenController < ApplicationController
```

Comment:
The idea to use the `AsyncInfoController` started because I originally used a `fetch` request (async in nature) so it used to "fit okay" in there. 

The advantage of using POST navigation (what happens whensubmitting a `form`) is that we get to reuse the logic from the `Omniauth<>Devise` integration. This includes things like `sign_in_and_redirect(@user, event: :authentication)`, flash messages, etc..

Because of this change ^ now the `AsyncInfoController` didn't fit quite well anymore. Also, looking into the future we could add more providers to this "Access Token based authentication". I'm thinking we could implement SIWA using Apple's JS SDK, which could ironically cleanup a bit the Omniauth logic.